src/commands/e2e/ConfigTemplate.ts

```diff
@@ -1,3 +1,5 @@
+import {E2E} from './E2E';
```

Comment:
Maybe we should try to switch to json again?? @rose-m

Reply:
Still not possible as we use `function() ....`